### PR TITLE
(DOCSP-11947), (DOCSP-40563), (DOCSP-41106): Updates for timestamps, procedures, and toggles.

### PR DESCRIPTION
**Pull request by corryroot**
# Pull Request Process

Please read the
[Style Guide Review Process](https://wiki.corp.mongodb.com/display/DE/Technical+Writer+Style+Guide+Review+Process)
wiki page.

Contributors should take the following actions:

- Request a PR review in #docs-style-guide Slack channel.
- Add the Style Guide team reps as reviewers.
- If any reviewer requests changes, address them and request another review.
- When you receive PR review approvals from all the Style Guide team reps,
  they will merge the PR and notify you of the merge in the Slack channel.

# Pull Request Description

I made the following small changes to the Style Guide to address a series of tickets in the backlog.

JIRA URLs:
- https://jira.mongodb.org/browse/DOCSP-11947
- https://jira.mongodb.org/browse/DOCSP-40563
- https://jira.mongodb.org/browse/DOCSP-41106

Staging URLs:

- [date](https://preview-mongodbcorryroot.gatsbyjs.io/meta/DOCSP-11947/style-guide/terminology/alphabetical-terms/#std-term-date)
- [timestamp](https://preview-mongodbcorryroot.gatsbyjs.io/meta/DOCSP-11947/style-guide/terminology/alphabetical-terms/#std-term-timestamp)
- [procedures](https://preview-mongodbcorryroot.gatsbyjs.io/meta/DOCSP-11947/style-guide/style/tasks/#procedures)
- [toggle](https://preview-mongodbcorryroot.gatsbyjs.io/meta/DOCSP-11947/style-guide/terminology/alphabetical-terms/#t:~:text=as%20two%20words.-,toggle,-Use%20toggle%20for)

Build: https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&jobId=66913e35fb8bd13b84567ec1

### CODE REVIEW

File: source/style-guide/includes/terms/d.rst [hunk 1]
```diff
@@ -31,6 +31,14 @@ D
    database
      Use *database* as one word.
 
+   date
```
Comment (jeff-allen-mongo) [inline source/style-guide/includes/terms/d.rst:34]: [Suggestion]

At the top of the Alphabetical List of Terms page, we have a table with icons for special types of terms. I think we should use the "Avoid" icon here like we do for "access (v)".

File: source/style-guide/includes/terms/t.rst [hunk 2]
```diff
@@ -140,11 +140,20 @@ T
...
+     instead of *date* wherever applicable to avoid confusion.
+
+     .. seealso:: 
+       
+        :term:`date`
 
    time zone
      Use *time zone* as two words.
 
+   toggle
```
Comment (jeff-allen-mongo) [inline source/style-guide/includes/terms/t.rst:153]: [Question]

Should we specify that this definition is referring to toggle as a noun?

File: source/style-guide/style/tasks.txt [hunk 3]
```diff
@@ -96,8 +96,12 @@ Procedures
...
 steps. Consider the following guidelines when creating a procedure:
 
--  If the procedure has more than one step, use a ``steps`` file for
-   the steps. Don't use bullets, except to list choices within a step.
+-  If the procedure has more than one step, use the ``.. procedure::`` 
+   directive for the steps. Don't use bullets, except to list choices 
+   within a step.
+-  If the procedure has more than one step, follow the guidance for 
+   introducing lists to introduce the procedure. To learn more, see 
+   :ref:`lists`.
```
Comment (jeff-allen-mongo) [inline source/style-guide/style/tasks.txt:104]: Not sure if it's within the scope of this PR, but a question came up on our team recently about how to handle "nested" steps, specifically whether nested steps should be denoted as `a. b. c.`, `i, ii, iii`, or other. Do we want to include guidance on that point here?

**Review by jeff-allen-mongo — APPROVED** [review]
Overall LGTM! Just have some non-blocking comments.

**Review by jeff-allen-mongo — APPROVED** [review]
LGTM! Thanks for incorporating the information on nested steps.

**Review by lindseymoore — APPROVED** [review]
Thanks, LGTM!

**Pull request merged**